external/artinis/ft_nirs_referencechannelsubtraction.m

```diff
@@ -0,0 +1,218 @@
+function data_rcs = ft_nirs_referencechannelsubtraction(cfg, datain)
+% FT_EXAMPLEFUNCTION demonstrates to new developers how a FieldTrip function should look like
```

Comment:
thanks for spotting this, I just fixed it.

[master c999e11] ENH - use correct function name in help
 4 files changed, 6 insertions(+), 2 deletions(-)
mbp> git push upstream master
Counting objects: 8, done.
Delta compression using up to 2 threads.
Compressing objects: 100% (8/8), done.
Writing objects: 100% (8/8), 703 bytes | 0 bytes/s, done.
Total 8 (delta 7), reused 0 (delta 0)
remote: Resolving deltas: 100% (7/7), completed with 7 local objects.
To git@github.com:fieldtrip/fieldtrip.git
   5a2b175..c999e11  master -> master